cg/meta/workflow/microsalt/constants.py

```diff
@@ -0,0 +1,4 @@
+from cg.constants.constants import FileExtensions
+
+
+QUALITY_REPORT_FILE_NAME: str = f"QC_done{FileExtensions.JSON}"
```

Comment:
Do we put anything inside this report? If so, I would not mind calling it "QC_report.json"

Reply:
I'll leave the name as is for now, might be some automation depending on this or something. Slightly paranoid